src/environments/environment.computas.ts

```diff
@@ -3,6 +3,6 @@ export const environment = {
   WS_ENDPOINT: 'wss://teknisk-museum-backend.azurewebsites.net',
   // TEKNISKBACKEND_ENDPOINT :'https://tekniskback.azurewebsites.net',
   TEKNISKBACKEND_ENDPOINT: 'https://teknisk-museum-backend-singleplayer.azurewebsites.net',
-  inactivityTime: 60 * 1000,
+  inactivityTime: 60 * 250, // 15 seconds

```

Comment:
Maybe change this to 15 * 1000 to easily see that it is 15 seconds times 1000 milliseconds in a second